docs/Operators.md

```diff
@@ -1376,11 +1473,10 @@ for i in range(len(shape)):
 node = onnx.helper.make_node(
     'Flatten',
     inputs=['a'],
-    outputs=['b'],
+    outputs=['b'], # Default value for axis: axis=1
 )
 
 shape = (5, 4, 3, 2)
-axis_default_value = 1
```

Comment:
This was actually a vestige from my previous PR https://github.com/onnx/onnx/pull/448 . 
Removed as suggested by @bddppq.